app/javascript/components/firmware-registry-add-edit.jsx

```diff
@@ -0,0 +1,135 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Grid, Row } from 'patternfly-react';
+import { componentTypes, validatorTypes } from '@data-driven-forms/react-form-renderer';
+import MiqFormRenderer from '../forms/data-driven-form';
+
+const FirmwareRegistryAddEdit = (props) => {
+  const TYPES = [
+    ['FirmwareRegistry::RestApiDepot', __('Rest API Depot')],
```

Comment:
I would prefer seeing this as an object. It will make it easier to use it later.